classes/Carrier.php

```diff
@@ -636,37 +636,45 @@ public static function getIdTaxRulesGroupMostUsed()
      */
     public static function getDeliveredCountries(int $id_lang, bool $active_countries = false, bool $active_carriers = false, $contain_states = null)
     {
-        $states = Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS('
-            SELECT s.*
-            FROM `' . _DB_PREFIX_ . 'state` s
-            ORDER BY s.`name` ASC');
+        $result = [];
 
-        $result = Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS('
-            SELECT cl.*,c.*, cl.`name` AS country, zz.`name` AS zone
-            FROM `' . _DB_PREFIX_ . 'country` c' .
-            Shop::addSqlAssociation('country', 'c') . '
+        $countries = Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS(
+            'SELECT cl.*,c.*, cl.`name` AS country, zz.`name` AS zone
```

Comment:
```suggestion
            'SELECT cl.*, c.*, cl.`name` AS country, zz.`name` AS zone
```

Reply:
I focused on testing the query, but I didn't notice the lack of spaces!

Reply:
Nah it's just my obsession with having a system in everything :D